Friendzymes/views/Libraries and Composites.csv

```diff
@@ -0,0 +1,36 @@
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Example Data,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Anderson library,Library specified as a list of 20 single parts,,True,pOpen_v4,,"J23100, J23101, J23102, J23103, J23104, J23105, J23106, J23107, J23108, J2309, J23110, J23111, J23112, J23113, J23114, J23115, J23116, J23117, J2318, J23119",,,,,,,,,,,,,,,,,,,,,,
+GFP expression cassette,RBS + CDS + terminator ,,False,,,B0032,E0040,B0015,,,,,,,,,,,,,,,,,,,,
+High/med/low plasmids,Plasmid library with 3 promoter options,,False,pSB1C3,,"J23101, J23106, J23117",GFP expression cassette,,,,,,,,,,,,,,,,,,,,,
+Interlab test strains,Library transformed into DH5-alpha,,True,,,High/med/low plasmids,,,,,,,,,,,,,,,,,,,,,,
+Interlab positive control,All-in-one specification of a transformed construct,,True,pSB1C3,,J23151,B0032,E0040,B0015,,,,,,,,,,,,,,,,,,,
+Interlab negative control,Transformation of a basic plasmid,,True,pSB1C3,,R0040,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Libraries and Composite Parts,Blue text column headers are optional,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Part/Library Name,Design Notes,Part Description,Final Product,Backbone/locus,Constraints,Part 1,Part 2,Part 3,Part 4,Part 5,Part 6,Part 7,Part 8,Part 9,,,,,,,,,,,,,,
+,,,False,,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
I see there's nothing in the composites sheet. Everything else is being ordered in a vector (generally pSB1C5). Right now, your sheet says that you want to have things delivered as just linear DNA fragments, which may not be compatible with FreeGenes processes. Is that the intention (in which case a discussion with @vinoo-igem is likely needed)? If it's not the intention, then the build plans should be expressed on this sheet and the "final product" markers on the parts sheet set to false.

Reply:
To make sure we understand, you're saying that the Libraries/Composites tab is where we should put information about the cloning/holding vector these parts should be stored in, correct? So currently the sheet implies no cloning vector, just raw DNA? 

I think it would be useful to discuss the relative merits of pSB1C5 vs pOpen_v3 vs pOpen_v4. Is there a thread where the engineering committee has covered this? We'd be open to pSB1C5 and would ideally like to use the same standard vector as the new iGEM Distribution; my main concern here is about compatibility with the existing FreeGenes libraries that are going into the Distro (e.g. Open Yeast Collection and the Protein Expression Toolkit), which to my knowledge use pOpen_v3.

Reply:
started an issue #244 to open up the discussion

Reply:
That's correct: you want to use the "Backbone/locus" column to indicate the vector holding the part. You should also consider whether you need to add flanking sequences, depending on whether the vector comes with them built in already.  

Which vectors can be used is a separate discussion with @vinoo-igem

Reply:
All FreeGenes parts, including Open Yeast Collection and Open Enzyme Collection, are clone by Twist in pOpen_v3. pOpen_v3 is ampR. All our parts should be cloned by Twist in this vector to make is useful as an AllClo/OYC part for GGA.